contracts/lease/src/lease/mod.rs

```diff
@@ -121,6 +117,12 @@ where
     LppLoan::Error: Into<ContractError>,
     Oracle: OracleTrait<Lpn>,
 {
+    pub const MIN_ASSET_AMOUNT: Amount = 10_000;
+    pub const MIN_SELL_ASSET_AMOUNT: Amount = 15_000_000;
```

Comment:
do not think we need these constants
inline them into the other pair ... and use `.into()` to shorter the initialization

Reply:
since MIN_ASSET and MIN_SELL_ASSET are constants, I can not initialize them using .into()